Its.Log/LogEntry.cs

```diff
@@ -353,7 +353,7 @@ public IEnumerable<object> Params
         /// If a log entry is part of a log activity, gets the objects passed to Confirm.
         /// </summary>
         [FormatterIgnores]
-        internal IEnumerable<object> Confirmations
+        public IEnumerable<object> Confirmations
```

Comment:
Why make this public?

Reply:
If I keep this as internal, at LogActivityTests line 501, 551, 605, 609, 612 I will see error 'LogEntry.Confirmations' is inaccessible due to its protection level.

It used to work before but I don't understand why.

Reply:
This was previously using `InternalsVisibleTo` which was in the deleted `AssemblyInfo.cs`. If you put back the IVTs then these various properties can be kept internal, which is preferable.